requirements/requirements.standalone.txt

```diff
@@ -279,7 +275,7 @@ pyyaml==5.4.1
     #   pulp-ansible
     #   pulpcore
     #   tablib
-redis==4.2.2
+redis==3.4.1
```

Comment:
i feel like we dont normally see these dep versions go backwards, might be worth looking at what pulpcore requires (maybe pulpcore started pinning it between 3.18.3 and 3.18.6)

Reply:
Hmm, that is indeed weird. Investigating.

Reply:
Looks like https://github.com/pulp/pulpcore/commit/157842021ea51613f052276caaef26ecabf60bda  changed the pulpcore redis dep from "redis>=3.4.0"  to "redis~=3.4.0"

Hopefully going backwards on redis doesn't break anything.